pkgs/development/libraries/libnftnl/default.nix

```diff
@@ -12,6 +12,19 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ libmnl ];
 
+  # Avoid
+  #
+  # # object.c:399:19: error: no member named '__builtin___snprintf_chk' in 'struct obj_ops'
+  # #                 ret = obj->ops->snprintf(buf + offset, remain, flags, obj);
+  preConfigure = lib.optionalString stdenv.cc.isClang ''
+    for f in $(find include src -type f | grep -E '\.(c|h)$'); do
+      substituteInPlace "$f" \
+        --replace snprintf op_snprintf
+    done
```

Comment:
You sure this isn't an exported symbol and we're not changing the ABI by doing this?

Reply:
_Also_, have you checked this can't be solved by simply doing `hardeningDisable = lib.optional stdenv.cc.isClang "fortify"`?

I would check but I'm unable to reproduce your problem - don't know how you're building things but `pkgsLLVM.libnftnl` builds for me without problems.

Reply:
Absolutely @risicle; thank you for your suggestion.

I have also added an extra check to make sure we are cross-compiling.